src/test/java/jalau/at18/architects/ResourceComparatorTest.java

```diff
@@ -0,0 +1,68 @@
+package jalau.at18.architects;
```

Comment:
Tests are good, but I have the feeling that implementation is a little bit more complicated than it needs, let me know if I'm missing something